app/src/main/contrib/components/Responsibilities/BurdenEstimates/UploadBurdenEstimatesPage.tsx

```diff
@@ -44,7 +44,8 @@ export class UploadBurdenEstimatesPage extends ContribPageWithHeader<UploadEstim
     }
 
     name() {
-        return "Upload burden estimates";
+        const r = responsibilityStore.getState();
+        return r.currentResponsibility.scenario.id;
```

Comment:
There's no way of distinguishing this from `DownloadCoveragePage` at the moment. Maybe use "Upload burden estimates for [SCENARIO]".